org-formation/060-cloudtrail/_tasks.yaml

```diff
@@ -12,6 +28,7 @@ CloudwatchTrailLog:
   Parameters:
     LogGroupName: !Sub '/aws/cloudtrail/${resourcePrefix}-cloudwatch-log.log'
     RetentionInDays: !GetAtt CurrentAccount.Tags.CloudwatchCloudTrailLogRetentionPeriod
+    KmsKeyId: !CopyValue [ !Sub '${primaryRegion}-${resourcePrefix}-cloudwatch-trail-kms-key-KeyArn' ]
```

Comment:
KeyArn?

Reply:
yup, docs says it can be an KMS ARN, alias or ID. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudtrail-trail.html#cfn-cloudtrail-trail-kmskeyid